nautobot_golden_config/models.py

```diff
@@ -196,23 +197,54 @@ def __str__(self):
         return f"{self.device} -> {self.rule} -> {self.compliance}"
 
     def save(self, *args, **kwargs):
-        """Perform that actual compliance check."""
+        """Performs the actual compliance check."""
         feature = {
             "ordered": self.rule.config_ordered,
-            "name": self.rule.feature.name,
-            "section": self.rule.match_config.splitlines(),
+            "name": self.rule,
         }
-        value = feature_compliance(feature, self.actual, self.intended, get_platform(self.device.platform.slug))
-        self.compliance = value["compliant"]
-        if self.compliance:
-            self.compliance_int = 1
+        if self.rule.config_type == ComplianceRuleTypeChoice.TYPE_JSON:
+            feature.update({"section": self.rule.match_config})
+            try:
+                json.loads(self.actual) and json.loads(self.intended)
```

Comment:
Are you using stringified JSON to keep a common model?

Reply:
Not at this time, intended and actual in this version of the plugin are to support both plain text and json.

Reply:
What I meant is the actual and intended attrs of the model are charfields so in a JSON diff your's able to store it as the same attr on the model as a text diff.

Reply:
> Are you using stringified JSON to keep a common model?

Yes